Hardcodet.NotifyIcon.Wpf/Source/NotifyIconWpf/Interop/BalloonFlags.cs

```diff
@@ -1,9 +1,12 @@
+using System;
+
 namespace Hardcodet.Wpf.TaskbarNotification.Interop
 {
     /// <summary>
     /// Flags that define the icon that is shown on a balloon
     /// tooltip.
     /// </summary>
+    [Flags]
```

Comment:
What is your reason to add the `Flags` attribute? You cannot combine them, after all :)

Reply:
But you are: https://github.com/hardcodet/wpf-notifyicon/blob/master/Hardcodet.NotifyIcon.Wpf/Source/NotifyIconWpf/TaskbarIcon.cs#L814

Reply:
Damn, I am! Still, I'm not sure this merits the flag - it's not your typical bit field... Thoughts?

Reply:
The Flags attribute is not necessary if you have correct values... https://stackoverflow.com/a/8480/920384

Reply:
I understand your struggle, the first 4 bits are more or less the type and combining them doesn't make sense. But the later values (0x10, 0x20, 0x80) are flags...

It would almost merit a separate struct or class.. :disappointed:

Reply:
I don't think users of the API will ever combine this - plus the attribute would imply that you can arbitrarily combine them (as with your typical `1,2,4,8,...` flags, which isn't the case here. So I'd skip it. OTOH, no point in spending too much time on this, we're probably neck deep in semantics now 😄

Reply:
Ok so we agree to remove the [Flags]?
I will add a comment for ReSharper to ignore the warning.

Reply:
@Lakritzator  cross-post... yeah, this is a weird one. otoh, it's the windows api, so let's keep together what doesn't really fit together ;)

Reply:
> 
> 
> Ok so we agree to remove the [Flags]?
> I will add a comment for ReSharper to ignore the warning.

I'd either remove it, or keep it with a comment that those can't be arbitrarily combined (may provide more context than just an R# ignore). I'll let you be the judge of it 👍

Reply:
It's not the weirdest Windows API case :laughing: 

Removed the flag, also found some more comments with typos or missing information.